pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -257,22 +278,25 @@ public RpcApi convert(final String name) throws RpcApisConversionException {
   )
   private final HostAndPort p2pHostAndPort = getDefaultHostAndPort(DefaultPeer.DEFAULT_PORT);
 
+  /** @deprecated Deprecated in favour of --chain option */
+  @Deprecated
   @Option(
+    hidden = true,
     names = {"--network-id"},
-    paramLabel = MANDATORY_INTEGER_FORMAT_HELP,
+    paramLabel = DefaultCommandValues.MANDATORY_INTEGER_FORMAT_HELP,
     description = "P2P network identifier (default: ${DEFAULT-VALUE})",
     arity = "1"
   )
   private final Integer networkId = null;
```

Comment:
Deprecating this at this point leaves us with no way to set the network-id for custom networks since it can't yet be specified in the genesis config.  It will work out in the end, but to keep with the idea of "master is always releasable" we should probably leave this as a fully supported option until we get support for it in the genesis config and then deprecate.

Reply:
this is just a first pass, I have to work this thing out.